docs/_plugins/shortcodes/repoStatus.cjs

```diff
@@ -17,9 +17,10 @@ function repoStatus({ heading = 'Repo status', type = 'Pattern' } = {}) {
   } else {
     return /* html*/`
 
-<section class="section section--palette-default container">
-  <a id="Component status"></a>
-  <h2 id="component-status" class="section-title pfe-jump-links-panel__section">${heading}</h2>
+<section class="section section--palette-default container">  
+  
+  ${`## ${heading} {.section-title .pfe-jump-links-panel__section} `}
```

Comment:
Would there ever be need to make this a different header?  `h3` etc

Doing something similar over here: https://github.com/RedHat-UX/red-hat-design-system/pull/1098/commits/bb5de95ddfde44f30eaf6e61cbace9caded58521

Reply:
@zeroedin I don't think so, but I can make the change to allow for it if we ever needed to

Reply:
@zeroedin do you know why L25 doesn't work, but L27 does? https://deploy-preview-1097--red-hat-design-system.netlify.app/elements/accordion/#repo-status

Reply:
because its markdown i believe it needs an empty line above it